loan-payment-skip/REPWRITERSPECS/BANNO.LOANPAYMENT.SKIP.V1.POW

```diff
@@ -437,17 +438,31 @@ PROCEDURE PROCGETPRELOADDATA
        IF LOANINELIGIBLECODE(LOANLOOP,TEMPLOOP)=TRUE AND
           INELIGIBLEDISPLAYED=FALSE THEN
         DO
-         PRINT Q+"ineligibilityReason"+Q+": "+Q+INELIGIBLEREASONS(TEMPLOOP)
-         IF TEMPLOOP=INELIGIBLECODELOANTYPE THEN
-          PRINT INVLNTYPE(LOANLOOP)
-         ELSE IF TEMPLOOP=INELIGIBLECODEACCTWARNING THEN
-          PRINT INVACCTWARNS(LOANLOOP)
-         ELSE IF TEMPLOOP=INELIGIBLECODELOANWARNING THEN
-          PRINT INVLNWARNS(LOANLOOP)
-         ELSE IF TEMPLOOP=INELIGIBLECODESERVICECODE THEN
-          PRINT EXISTINGLNSERVS(LOANLOOP)
-         PRINT Q
-         NEWLINE
+         IF TEMPLOOP=INELIGIBLECODELOANTYPE OR

```

Comment:
I think you can simplify this quite a bit.  Something like this:
```
         INELIGDETAIL=""
         IF TEMPLOOP=INELIGIBLECODELOANTYPE THEN
          INELIGDETAIL=INVLNTYPE(LOANLOOP)
         ELSE IF TEMPLOOP=INELIGIBLECODEACCTWARNING THEN
          INELIGDETAIL=INVACCTWARNS(LOANLOOP)
         ELSE IF TEMPLOOP=INELIGIBLECODELOANWARNING THEN
          INELIGDETAIL=INVLNWARNS(LOANLOOP)
         ELSE IF TEMPLOOP=INELIGIBLECODESERVICECODE THEN
          INELIGDETAIL=="Existing service codes = "+EXISTINGLNSERVS(LOANLOOP)

         PRINT Q+"ineligibilityReason"+Q+": "+Q+INELIGIBLEREASONS(TEMPLOOP)+Q+","
         NEWLINE
         PRINT Q+"ineligibilityDetail"+Q+": "+Q+INELIGDETAIL+Q
         NEWLINE
         INELIGIBLEDISPLAYED=TRUE
```

Reply:
This removes some duplicate code.  It is fine to just have an empty `ineligibilityDetail` for those reasons that don't have a detail.

Reply:
Thanks for cleaning that up @rrobison44 . I've made the change and tested that it work. How does the code look now?